### ISSUE TITLE
Spurious warnings generated in flake8-bugbear

### ISSUE THREAD

**Issue by dopplershift** [gforcada/flake8-builtins#35]
Once I upgrade to flake8-builtins 1.4.0, I started seeing these errors:
```
metpy/calc/thermo.py:2024:28: B007 Loop control variable 'ret' not used within the loop body. If this is intended, start the name with an underscore.
```
These are actually generated by the flake8-bugbear (18.2.0) plugin, and are spurious. Installing flake8-builtins 1.3.1 causes them to go away, so I'm guessing the problem actually lies in flake8-builtins.

**Comment by gforcada** [comment]
@dopplershift I saw them as well, but only on 3.5 and restarting that single specific job did the trick, I did not dig it further as it was going away on every second run...

**Comment by dopplershift** [comment]
Running it locally, I can reproduce at will on Python 3.6. I'll try to dig into it more in a few days (need to get a release out first).

**Comment by gforcada** [comment]
With #37 being merged and released as 1.4.1 this should be fixed, would you mind giving it a try?

**Comment by dopplershift** [comment]
I can confirm this is fixed with 1.4.1. Thanks!
